server/src/main/java/org/opensearch/cluster/routing/allocation/DiskThresholdSettings.java

```diff
@@ -97,6 +97,12 @@ public class DiskThresholdSettings {
         Setting.Property.Dynamic,
         Setting.Property.NodeScope
     );
+    public static final Setting<Boolean> CLUSTER_CREATE_INDEX_BLOCK_AUTO_RELEASE = Setting.boolSetting(
+        "cluster.blocks.create_index.autorelease",
```

Comment:
nit : `cluster.blocks.create_index.auto_release `

Reply:
Addressed the comments.